librz/type/base.c

```diff
@@ -37,7 +37,7 @@ RZ_API RZ_BORROW RzBaseType *rz_type_db_get_base_type(const RzTypeDB *typedb, RZ
 	bool found = false;
 	RzBaseType *btype = ht_pp_find(typedb->types, name, &found);
 	if (!found || !btype) {
-		eprintf("Cannot find base type \"%s\"\n", name);
+		//eprintf("Cannot find base type \"%s\"\n", name);
```

Comment:
I suppose it's a temporary change?

Reply:
Yes, it's for debugging. There are so many of it so I disabled it.